.eslintignore

```diff
@@ -2,7 +2,7 @@
 /coverage
 /node_modules
 /tests/fixtures
-/tests/integrations/*/node_modules
+/tests/integrations/eslint-plugin-import
```

Comment:
EsLint 6.7.0 changes eslintignore parsing.  The previous one seems to have ignored the whole folder which I'm now explicitly doing.  Otherwise, you get errors about eslint-plugin-ignore not being installed when running `npm run lint` after a clean install.